From 6baca260d11b4c2f92e8765ff0f66786e507022a Mon Sep 17 00:00:00 2001
From: Dan Williams <dcbw@redhat.com>
Date: Tue, 26 May 2015 18:21:31 -0500
Subject: Partially revert "dhcp: gracefully recover from failed DHCP BOUND
 state transitions (bgo #743700)"

This reverts commit 7daf63461de4195b1626ca15f835fc7cbc56e847.

Turns out the removal of the second set of [] in configure.ac causes the command
to be wrong in 'configure' and the test to be incorrect.

diff --git a/configure.ac b/configure.ac
index ba22057..d9a90c1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -668,10 +668,10 @@ fi
 if test "$with_dhcpcd" = "yes"; then
 	AC_PATH_PROGS(with_dhcpcd, dhcpcd, no, /sbin:/usr/sbin:/usr/local/sbin)
 	if test "$with_dhcpcd" != "no"; then
-		if ! $with_dhcpcd --version 2>&1 | grep -q "^dhcpcd [456789]\."; then
+		if ! $with_dhcpcd --version 2>&1 | grep -q "^dhcpcd [[456789]]\."; then
 			AC_MSG_WARN([Cannot use dhcpcd, version 4.x or higher is required])
 			with_dhcpcd=no
-		elif $with_dhcpcd --version 2>&1 | grep -q "^dhcpcd [6789]\."; then
+		elif $with_dhcpcd --version 2>&1 | grep -q "^dhcpcd [[6789]]\."; then
 			AC_DEFINE(DHCPCD_SUPPORTS_IPV6, 1, [Define if dhcpcd supports IPv6 (6.x+)])
 		fi
 	fi
-- 
cgit v0.10.2

